web/src/test/java/com/replica/StreetEdgeExporterTest.java

```diff
@@ -0,0 +1,95 @@
+package com.replica;
+
+import com.google.common.base.Preconditions;
+import com.graphhopper.config.Profile;
+import com.graphhopper.http.GraphHopperApplication;
+import com.graphhopper.http.GraphHopperBundle;
+import com.graphhopper.http.GraphHopperServerConfiguration;
+import com.graphhopper.http.cli.ExportCommand;
+import com.graphhopper.http.cli.ImportCommand;
+import com.graphhopper.replica.StreetEdgeExporter;
+import com.graphhopper.util.Helper;
+import io.dropwizard.cli.Cli;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import io.dropwizard.util.JarLocation;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+
+import org.apache.commons.csv.CSVRecord;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class StreetEdgeExporterTest {
+
+    private static final String TARGET_DIR = "./target/gtfs-app-gh/";
+    private static final String TRANSIT_DATA_DIR = "transit_data/";
+    public static final DropwizardAppExtension<GraphHopperServerConfiguration> app = new DropwizardAppExtension<>(GraphHopperApplication.class, createConfig());
+    private static GraphHopperServerConfiguration createConfig() {
+        GraphHopperServerConfiguration config = new GraphHopperServerConfiguration();
+        config.getGraphHopperConfiguration().
+                putObject("graph.flag_encoders", "car,foot").
```

Comment:
This line is the only thing that differs between this setup code and the equivalent in the other test. I specified "car,foot" here, as well as in the yaml config file. Totally not clear to me why we do it in both places but 🤷‍♂️

Reply:
Ahh, I see - in the other test, we actually use `app`; in this test, we copied over the same structure, but we never actually use this `app` variable. When we actually call the CLI commands below we're just passing the name of the config yaml, so I think the yamls are actually all we need, and this code at the beginning of this test can be removed

Reply:
Ah, thanks for looking at this with fresh eyes. When I was first trying to get this off the ground I was having the hardest time getting things to run / setup correctly, and so I copied over as much as I could. Now that I have things working, it's clearer that I can remove this section. Will include with my next phase of this work, thanks!